File: coordinate_geometry.cpp
#include <iostream>
#include <cmath>

using namespace std;
int main() {
    int p1x;
    int p1y;
    int p2x;
    int p2y;

    cout << "Enter p1x: " << endl;
    cin >> p1x;
    cout << "Enter p1y: " << endl;
    cin >> p1y;
    cout << "Enter p2x: " << endl;
    cin >> p2x;
    cout << "Enter p2y: " << endl;
    cin >> p2y;

    double xdiff = pow((p2x - p1x), 2); ;
    double ydiff = pow((p2y - p1y), 2); ;
    double distance = sqrt(xdiff + ydiff);
    cout << endl << distance << endl;
}

File: coordinate_geometry.py
import math

p1x = input("Enter the first pints x value:")
p1y = input("Enter the first points y value:")
p2x = input ("Enter the second points x value:")
p2y = input ("Enter the second points y value:")
p1x = int(p1x)
p1y = int(p1y)
p2x = int(p2x)
p2y = int(p2y)

xDif = (p1x - p2x)**2
yDif = (p1y - p2y)**2

distance = math.sqrt((xDif + yDif))
print(distance)